fcl/prolog_trigger.fcl

```diff
@@ -14,7 +14,7 @@ TrkAnaTrigger : {
     FillCaloMC : false
     RecoCountTag : ""
     FillTriggerInfo : true
-    FillTrkQualInfo : false
+    //FillTrkQualInfo : false
```

Comment:
You can just remove this line - the parameter no longer exists. I missed deleting it here when doing the recent TrkQual update